src/components/border_editor/border_editor.xml

```diff
@@ -0,0 +1,81 @@
+<templates>
+  <t t-name="o-spreadsheet-BorderEditor" owl="1">
+    <Popover t-props="popoverProps">
+      <div
+        class="o-border-selector"
+        t-on-click.stop=""
+        t-att-class="props.class ? props.class : ''">
+        <div class="o-border-selector-section">
+          <div
+            t-foreach="BORDER_POSITIONS"
+            t-as="borderPositionsRow"
+            t-key="borderPositionsRow"
+            class="o-dropdown-button o-dropdown-line">
+            <span
+              t-foreach="borderPositionsRow"
+              t-as="borderPosition"
+              t-key="borderPosition"
+              class="o-line-item o-tool"
+              t-att-class="{active:currentBorderPosition == borderPosition[0], [props.itemsClass] : true}"
+              t-att-name="borderPosition[0]"
+              t-on-click.stop="() => this.setBorderPosition(borderPosition[0])">
+              <t t-call="{{borderPosition[1]}}"/>
+            </span>
+          </div>
+        </div>
+        <div class="o-divider"/>
+        <div class="o-border-selector-section">
+          <div
+            class="o-tool m-0 p-0 o-dropdown o-with-color"
+            t-att-class="props.itemsClass ? props.itemsClass : ''"
+            title="Border color"
+            t-on-click.stop="(ev) => this.toggleDropdownTool('borderColorTool', ev)">
+            <ColorPickerWidget
+              currentColor="props.currentBorderColor"
+              toggleColorPicker="(ev) => this.toggleDropdownTool('borderColorTool', ev)"
+              showColorPicker="state.activeTool === 'borderColorTool'"
+              onColorPicked="(color) => this.setBorderColor(color)"
+              title="env._t('Border Color')"
+              icon="props.currentBorderColor === '' ? 'o-spreadsheet-Icon.BORDER_NO_COLOR' : 'o-spreadsheet-Icon.BORDER_COLOR'"
+              dropdownMaxHeight="this.props.dropdownMaxHeight"
+              class="'o-tool o-dropdown-button o-border-color-picker-button'"
+            />
+            <t t-call="o-spreadsheet-Icon.TRIANGLE_DOWN"/>
+          </div>
+          <div
+            class="o-tool o-border-style-tool o-dropdown"
+            t-att-class="props.itemsClass ? props.itemsClass : ''"
+            title="Line style"
+            t-ref="lineStyleButton"
+            t-on-click.stop="(ev) => this.toggleDropdownTool('borderTypeTool', ev)">
+            <t t-call="o-spreadsheet-Icon.BORDER_TYPE"/>
+            <t t-call="o-spreadsheet-Icon.TRIANGLE_DOWN"/>
+            <Popover
+              t-props="lineStylePickerPopoverProps"
+              t-if="state.activeTool === 'borderTypeTool'">
+              <div class="o-border-style-dropdown">
+                <t t-foreach="borderStyles" t-as="borderStyle" t-key="borderStyle">
+                  <div
+                    t-if="borderStyle !== 'double'"
```

Comment:
then why include `"double"` in `borderStyles` ?

Reply:
A simple mistake. At first I was thinking to introduce it, but it was finally let aside (for the moment at least)